src/commands/kv/namespace/list.rs

```diff
@@ -1,25 +1,47 @@
+extern crate serde_json;
+
 use cloudflare::endpoints::workerskv::list_namespaces::ListNamespaces;
+use cloudflare::endpoints::workerskv::list_namespaces::ListNamespacesParams;
```

Comment:
these two lines, that import `ListNamespaces` and `ListNamespacesParams`, need to come from the dubs crate.